typespec-tests/src/main/java/com/cadl/flatten/FlattenAsyncClient.java

```diff
@@ -132,6 +135,47 @@ public Mono<Response<Void>> sendLongWithResponse(String id, BinaryData request,
         return this.serviceClient.sendLongWithResponseAsync(id, request, requestOptions);
     }
 
+    /**
+     * The update operation.
+     * <p><strong>Request Body Schema</strong></p>
+     * <pre>{@code
+     * {
+     *     patch (Optional, Required on create): {
+     *         title: String (Optional)
+     *         description: String (Optional)
+     *         status: String(NotStarted/InProgress/Completed) (Optional)
+     *     }
+     * }
```

Comment:
Payload is in this shape ("patch" is just a property in request body), flatten or not.